packages/@okta/vuepress-site/docs/guides/oie-embedded-common-download-setup-app/main/go/githubinstructions.md

```diff
@@ -1,8 +1,12 @@
-* To use the sample applications, clone the
-  [samples-golang](https://github.com/okta/samples-golang) repository
-  to your local hard drive and follow the instructions in the readme
-  to get them running locally.
-* To checkout or contribute to the SDK source code, clone the
-  [okta-idx-golang](https://github.com/okta/okta-idx-golang)
-  repository to your local hard drive, make your changes, and
-  submit pull requests.
+* To use the sample applications, clone the [samples-golang](https://github.com/okta/samples-golang) repository to your local directory. You run the sample app later after you [set the configuration values](/docs/guides/oie-embedded-common-download-setup-app/-/main/#set-the-configuration-values).
```

Comment:
It seems slightly odd to me that this section starts with a bulleted list. Maybe un-bullet the first two paragraphs?